classification1.Rmd

```diff
@@ -472,86 +489,155 @@ perim_concav + annotate("path",
 ### More than two explanatory variables 
 
 Although the above description is directed toward two predictor variables, 
-exactly the same K-nearest neighbours algorithm applies when you
+exactly the same $K$-nearest neighbors algorithm applies when you
 have a higher number of predictor variables (i.e., a higher-dimensional
 predictor space).  Each predictor variable may give us new
 information to help create our classifier.  The only difference is the formula
-for the distance between points. In particular, let's say we have $m$ predictor
+for the distance between points. 
+
+Suppose we have $m$ predictor
 variables for two observations $a$ and $b$, i.e., 
 $a = (a_{1}, a_{2}, \dots, a_{m})$ and
 $b = (b_{1}, b_{2}, \dots, b_{m})$.
-Previously, when we had two variables, we added up the squared difference between each of our (two) variables,
-and then took the square root. Now we will do the same, except for *all* of our
-$m$ variables.  In other words, the distance formula becomes
+
+The distance formula becomes
 
 $$\mathrm{Distance} = \sqrt{(a_{1} -b_{1})^2 + (a_{2} - b_{2})^2 + \dots + (a_{m} - b_{m})^2}.$$
 
 This formula still corresponds to a straight-line distance, just in a space with 
-more dimensions. For example, suppose we decide to use 3 predictor variables (so, a 3-dimensional space):
-perimeter, concavity, and symmetry. Figure \@ref(fig:05-more) shows what
-the data look like when we visualize them as a 3-dimensional scatter.
+more dimensions. Suppose we want to calculate the distance between a new observation 
+that we want to classify with a perimeter of 0, concavity of 3.5 and symmetry of 1 and
+another observation with a perimeter, concavity and symmetry of 0.417, 2.31 and 0.837 respectively. 
+We have three predictor variables, perimeter, concavity, and symmetry (so, a 3-dimensional space) for two observations where
+$\mathrm{observation}_{old} = (0.417, 2.31, 0.837)$ and $\mathrm{observation}_{new} = (0, 3.5, 1)$.
```

Comment:
this latex needs some work

in fact, I would just rewrite this to entirely get rid of \mathrm{observation}_{old}. No need to make named observations here